src/styles/overlay-panel.less

```diff
@@ -16,6 +16,7 @@
   border: 1px solid @color-pf-black-400;
   height: 50%;
   left: 20px;
+  min-height: 350px;
```

Comment:
This is fine for now but I believe UX has plans to specify two settings for the overlay panel height. Callers can choose either large or small to set the height/max-height.